docs/source/Appendix_A.md

```diff
@@ -186,30 +186,35 @@ behavior of certain value classes (for example the `numericClass` value class).
 * - Value class
   - Allowed characters
 * - dateTimeClass
-  - `digits`  `T`  `:`  `-` 
+  - `digits`,  `colon`,  `hyphen`, `period`, `uppercase`
 * - nameClass
-  - `alphanumeric`  `-`  `_`
+  - `alphanumeric`, `blank`, `hyphen`, `period`, `underbar`, `non-ascii` 
 * - numericClass
-  - `digits`  `.`  `-`  `+`  `E`  `e` 
+  - `digits`,  `period`,  `hyphen`,  `plus`, 'caret',  `E`,  `e` `

```

Comment:
`'caret'` is single-quoted instead of back-ticked, and there's an extra backtick at the end.

Reply:
Corrected.